src/components/cards/TappableCard.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+
+import * as React from 'react'
+import { TouchableOpacity, View } from 'react-native'
+import FontAwesome5 from 'react-native-vector-icons/FontAwesome5'
+
+import { memo } from '../../types/reactHooks.js'
+import { type Theme, cacheStyles, useTheme } from '../services/ThemeContext.js'
+import { Card } from './Card'
+
+/**
+ * An (optionally) tappable card that displays its children in up to two left/right
+ * sections. If the card is configured to be tappable, a chevron is drawn on the
+ * right side of the card.
+ */
+const TappableCardComponent = ({
+  leftChildren,
+  nonTappable = false,
+  onPress,
+  rightChildren
+}: {
+  leftChildren: React.Node,
+  nonTappable?: boolean,
```

Comment:
Remove the negative, just `isTappable` which defaults to `true`.

Reply:
Set it up this way because our other components seem to do this. 

I believe it allows flags to be easily set when calling the component with `<Component negativeFlag>` instead of `<Component flag={false}>`